.graphqlrc.yml

```diff
@@ -0,0 +1,9 @@
+projects:
+  # 'app-core':
+  #   schema: ['https://api.boson.health/graphql']
+  #   documents: ['apps/app/**/*.{graphql,js,ts,jsx,tsx}']
+
+  'app-v2':
+    schema: ['http://clinical-api.tau.health:8080/graphql']
+    documents: ['apps/app/**/SettingsV2/**/*.{ts,tsx}']
+    extensions: { languageService: { cacheSchemaFileForLookup: true } }
```

Comment:
Gotta figure out a sane approach to this so that we _can_ point locally, or _can_ point remotely.

Reply:
Yeah. The issue is that this config is used by the VSCode syntax highlighter...